src/main/java/codeu/controller/UsersServlet.java

```diff
@@ -0,0 +1,19 @@
+package codeu.controller;
```

Comment:
I think we can just use the existing ProfileServlet.java to sets some attributes to the request object when no username is specified (after the slash e.g. users/username) so that the profile.jsp page shows list of all users of the site. Otherwise, ProfileServlet.java sets the attributes so that the user's profile shows up. Once everything works, change the name ProfileServlet.java to UsersServerlet.java, and name profile.jsp to users.jsp. Or something like that.